spock-outputcapture/src/main/groovy/io/github/joke/spockoutputcapture/CapturedOutputImpl.groovy

```diff
@@ -15,7 +15,7 @@ class CapturedOutputImpl implements CapturedOutput {
 
     @Override
     List<String> getLines() {
-        buffer.toString().split('\n')
+        buffer.toString().split('(\r\n|\n|\r)')
```

Comment:
Could you elaborate the use case a little bit.
Maybe the split should be done on
```System.lineSeparator()```

Reply:
Hi @joke , thanks for the quick reaction!

In my case, I was working under Windows, i.e. with `\r\n` line endings. Your current implementation then obviously left a `\r` at the end of every line which caused my RegEx to fail.

Then I used `System.lineSeparator()` to split, which _almost_ solved the problem. But I noticed that some developers occasionally put manual line breaks in their log output to increase readability. And for this purpose, they tend to use `\n` instead of `System.lineSeparator()`.

As the _intention_ was a line break, I assumed that spock-outputcapture should regard every kind of line break to start a new line.

Reply:
Sounds reasonable.